inference-engine/samples/common/samples/args_helper.hpp

```diff
@@ -85,3 +85,240 @@ void parseInputFilesArguments(std::vector<std::string> &files) {
         readInputFilesArguments(files, args.at(i));
     }
 }
+
+namespace {
+
+void splitStringList(const std::string& str, std::vector<std::string>& out, char delim) {
+    out.clear();
```

Comment:
Why this function doesn't return result? Is any reason why `out` is pass as parameter and clear it next line?